Tools/gazebo_sitl_multiple_run.sh

```diff
@@ -27,7 +33,7 @@ function spawn_model() {
 	pushd "$working_dir" &>/dev/null
 	echo "starting instance $n in $(pwd)"
 	../bin/px4 -i $n -d "$src_path/ROMFS/px4fmu_common" -w sitl_${MODEL}_${n} -s etc/init.d-posix/rcS >out.log 2>err.log &
-	python3 ${src_path}/Tools/sitl_gazebo/scripts/xacro.py ${src_path}/Tools/sitl_gazebo/models/rotors_description/urdf/${MODEL}_base.xacro \
+	python3 ${src_path}/Tools/sitl_gazebo/scripts/xacro.py ${src_path}/Tools/sitl_gazebo/models/rotors_description/urdf/${ARR_MODEL[0]}_base.xacro \
```

Comment:
This will break for `standard_vtol` since you are truncating the `_` and passing only `standard`

Reply:
Thanks for your good advice!

How can I fix it? Do you have any idea?

Reply:
You can fix it by using MODEL instead of ARR_MODEL right? I do you need to define ARR_MODEL?

Reply:
Yes.. if I add xacro for iris_rtps, it can be solved simply. However, I think the xacro file will be duplicated. In addition, other models will have the same problem if the model is added like iris_stereo_camera ...
How do you think about it?

Reply:
Yes - but as of now `iris` and `iris_rtps` ARE different models, therefore it should be handled consistently. If we want to introduce better configuration management, we should do it for all models together throughout the code base, not just here.

Reply:
How about checking base models like below?

```bash
SUPPORTED_MODELS=("iris" "plane" "standard_vtol")
for bm in ${SUPPORTED_MODELS[@]}
do
    if [ $(echo $MODEL | grep -F $bm -c) != 0 ] 
    then
        base_model=$bm
    fi  
done
```

Reply:
@stmoon the `SUPPORTED_MODELS` list should only contain models that are supported. The base model searching is something we want to support, but not here. For example with the script above, the script will not fail if you pass `iris_opt_flow` but then it will wrongly start the `iris` model. Therefore your suggestion ONLY works for `iris_rtps` and none of the other models

Reply:
@Jaeyoung-Lim  please check it (https://github.com/PX4/sitl_gazebo/pull/558).